### ISSUE TITLE
[Dúvida] - LoteCobV - Limite de Registro de Cobranças em lote

### ISSUE THREAD

**Issue by masyaf** [bacen/pix-api#258]
Prezados, bom dia.

Gostaria de saber se há algum limite previsto no registro de cobranças em lote.
Pergunto isto, pois é algo comum os webservices limitarem o envio de lotes XPTO por quantidade de ocorrências, exemplo; 100/200 cobranças, por lote.

Olhei no manual de Iniciação Pix e na Espec da API, não encontrei nada referente à isto.

Hoje, nós temos clientes que emitem cerca de 30 mil faturas de mensalidades por mês, e é bem provável que existam lotes de registro de cobranças de no mínimo 5 mil cobranças por lote, por exemplo.

**Comment by rubenskuhl** [comment]
Em outras chamadas da API o limite de paginação é de 100 transações. Mas acho que esse pode ser o número mínimo e o PSP ter possibilidade de definir um número maior, com o requisito de que esse número seja informado aos clientes da API.

**Comment by masyaf** [comment]
@rubenskuhl  como ainda não há nada oficial acerca deste assunto pelo BACEN, então muito provavelmente irá ficar à cargo de cada PSP definir, de acordo com sua infraestrutura? Será?

**Comment by rubenskuhl** [comment]
> @rubenskuhl como ainda não há nada oficial acerca deste assunto pelo BACEN, então muito provavelmente irá ficar à cargo de cada PSP definir, de acordo com sua infraestrutura? Será?

Sim, mas acho que um condicionamento leve do BACEN neste caso (definindo um mínimo mas não um máximo) seja interessante. @ninrod @dmkamers

**Comment by ninrod** [comment]
Não colocamos limites de propósito para deixar cada PSP estabelecer seu próprio limite. O limite mínimo, acredito que é irrelevante neste caso. 

O limite máximo é mais complicado estabelecermos via API. O PSP A poderia estabelecer 10 milhões de limite máximo enquanto o PSP poderia estabelecer 10k. Não achamos que isso é um grande problema é dá alguma flexibilidade em termos de diferenciação. 

Se acharem que isso é um problema, podem colocar aqui suas razões e vamos analisar.

**Comment by rubenskuhl** [comment]
Uma vantagem do limite mínimo é permitir uma base comum, e aí um desenvolvedor poderia ou usar essa base comum ou personalizar. Mas é uma vantagem pequena. 

O que porém acho que precisa ser exigido é que o PSP informe se, caso suporte o escopo lotecobv, qual é o limite dele. Seria inclusive um dado interessante para o OpenBanking: escopos suportados, limites numéricos etc.
